python_modules/dagster/dagster/_core/storage/event_log/base.py

```diff
@@ -309,6 +309,9 @@ def get_maximum_record_id(self) -> Optional[int]:
     def can_cache_asset_status_data(self) -> bool:
         pass
 
+    def can_write_asset_cached_status_data(self) -> bool:
+        return True
```

Comment:
I thought this was what `can_cache_asset_status_data` was - these probably need comments to explain the difference 

any risk in having it `True` on the base instead of pushing the impl to subclasses?